natgeo/mortar
# Feature/mortar 60

https://jira.nationalgeographic.com/browse/MORTAR-60

Testing Instructions 

-  Git checkout feature/mortar-60
- Gulp serve
- Check the code and make sure the font size has been updated for .mt_body 
- Visit http://localhost:3000/theme/
